libs/partners/mistralai/langchain_mistralai/chat_models.py

```diff
@@ -283,9 +283,16 @@ def _convert_message_to_mistral_chat_message(
                 tool_calls.append(chunk)
         else:
             pass
+        if tool_calls and message.content:
+            # Assistant message must have either content or tool_calls, but not both.
```

Comment:
Mistral raises an error with this message if you try to pass both. Not sure if there's a good solution to this.